.github/workflows/release-code-freeze.yml

```diff
@@ -0,0 +1,145 @@
+name: "Release - Code freeze"
+
+# This action will run according to the cron schedule or when triggered manually
+on:
+  schedule:
+    - cron: '0 12 * * 0' # Run at 1200 UTC on Sundays.
+  workflow_dispatch:
+    inputs:
+      skipSlackPing:
+        description: "If true, the Slack ping will be skipped"
+        type: boolean
+        required: false
+        default: false
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  check-code-freeze:
+    name: "Check that today is the day of the code freeze"
+    runs-on: ubuntu-20.04
+    outputs:
+      freeze: ${{ steps.check-freeze.outputs.FREEZE }}
+      releaseVersion: ${{ steps.next-version.outputs.RELEASE_VERSION }}
+      releaseDate: ${{ steps.next-version.outputs.RELEASE_PLANNED_DATE }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+#        with:
```

Comment:
Can we remove these commented lines if not using them?

Reply:
Yes, but rather uncomment it (see previous comment [here](https://github.com/Automattic/woocommerce-payments/pull/4925#discussion_r997854804))

Reply:
I uncommented this part. We want the code freeze workflow to execute from the `develop` branch